app/models/card.rb

```diff
@@ -1,2 +1,12 @@
+class CompareValidator < ActiveModel::Validator
+  def validate(record)
+    if record.original_text.downcase  == record.translated_text.downcase
```

Comment:
Unnecessary spacing detected.<br>Operator `==` should be surrounded with a single space.